app/sbna-poc/v1_0__initial-form/fields/finance.ts

```diff
@@ -402,7 +359,7 @@ export const baseFinanceFields: Array<FormWizard.Field> = [
   {
     text: 'Is [subject] affected by debt?',
     code: 'finance_debt',
-    type: FieldType.Radio,
+    type: FieldType.CheckBox,
```

Comment:
Same as above, this will require `multiple: true` to allow an array of values to be submitted